### DISCUSSION TITLE
The resources do not show in filament sidebar.

### DISCUSSION THREAD

**Discussion by sumer5020** [filamentphp/filament#5735] [Help] [answered]
I Try create resources for my model with name Dept that named in DB Depts,
I do like this to create my resource:
`php artisan make:filament-resource Dept --soft-deletes`

the resources created with empty table, and it's not shown in filament sidebar.
Note that: only the User model and the models that have complex name created and shown successfully, any other models with single word name not shown in filament sidebar.

how to fix this issue?

**Comment by danharrin** [comment, marked as answer]
Do you have a policy for this Rule model?

**Reply by sumer5020** [reply]
> Do you have a policy for this Rule model?

No, there is no policy yet.

**Reply by ManojKiranA** [reply]
Can you reproduce it with fresh installation

**Reply by sumer5020** [reply]
The problem solved by using old version.

**Reply by danharrin** [reply]
Thats not a solution, theres something wrong in your code otherwise everyone else would be reporting the same thing?

**Reply by sumer5020** [reply]
Of course, the problem is not with the version of Filament, but with the compatibility of the version with the rest of the components of the current application.

**Comment by josefbehr** [comment]
Why is the resource named "Rule" when the table and model are named "Dept"? How should the command know which model to create the resource for? Or am I misunderstanding?

**Reply by sumer5020** [reply]
No, I have more than one Model with the same problem [Rule - Dept - Etc]
In DataBase this will looks like [rules - depts - etc].

it's `php artisan make:filament-resource Dept --soft-deletes`
I fixed the question to be clear.

**Comment by eafarooqi** [comment]
I have the same problem. Fresh installation with laravel latest version. Fresh laravel and fresh filament. just created a simple resource name City and its not showing in the sidebar. I have also tried with the customer as mentioned in the documentation, still nothing in the sidebar and there is no Policy

php artisan make:filament-resource City
php artisan make:filament-resource Customer

**Reply by musilondrej** [reply]
To resolve the issue where Filament resources aren’t appearing in the sidebar try the following steps:
1. `php artisan filament:optimize-clear`
2. Delete the cache folder manually: If clearing the cache doesn’t work, try manually deleting the Filament cache folder located at bootstrap/cache/filament.

This should refresh the Filament dashboard and ensure that the resources are correctly displayed in the sidebar.

**Reply by tahinuralam** [reply]
> To resolve the issue where Filament resources aren’t appearing in the sidebar try the following steps:
> 1. `php artisan filament:optimize-clear`
> 2. Delete the cache folder manually: If clearing the cache doesn’t work, try manually deleting the Filament cache folder located at bootstrap/cache/filament.
> 
> This should refresh the Filament dashboard and ensure that the resources are correctly displayed in the sidebar.
> 

Thanks dear it works fine

**Reply by Erickhrs** [reply]
> I have the same problem. Fresh installation with laravel latest version. Fresh laravel and fresh filament. just created a simple resource name City and its not showing in the sidebar. I have also tried with the customer as mentioned in the documentation, still nothing in the sidebar and there is no Policy
> 
> php artisan make:filament-resource City php artisan make:filament-resource Customer

same here, it´s not displaying City resource

**Comment by Erickhrs** [comment]
I also have this problem when  I  create a source called City in a fresh filament... (php artisan make:filament-resource City)
